src/components/ProductBarInfoPage.jsx

```diff
@@ -30,6 +30,10 @@ const useStyles = makeStyles({
     height: "100%",
     overflow: "hidden",
   },
+  description: {
+    maxWidth: "700px",
```

Comment:
I confirmed this maxWidth value with Sophia :)

Reply:
Great thanks for confirming!